src/pages/team/[teamId].astro

```diff
@@ -21,6 +21,11 @@ const socialNetworksWithIcon = socialNetworks.map((href) => ({
 const president = await findPresidentBy({ id: presidentId })
 const playersTwelve = await getPlayersTwelveFor({ teamId: id })
 
+const porteros = players.filter((player) => player.role === 'Portero')
+const defensas = players.filter((player) => player.role === 'Defensa')
+const medio = players.filter((player) => player.role === 'Medio')
+const delantero = players.filter((player) => player.role === 'Delantero')
```

Comment:
I think it would be better to leave the variables in English so as not to break the scheme
```suggestion
const goalkeepers = players.filter((player) => player.role === 'Portero')
const defenses = players.filter((player) => player.role === 'Defensa')
const medium = players.filter((player) => player.role === 'Medio')
const forward = players.filter((player) => player.role === 'Delantero')
```